### DISCUSSION TITLE
챕터 6. 캐시 최적화

### DISCUSSION THREAD

**Discussion by Tanney-102** [Frontend-Divers/performance-optimization#31] [함께 얘기하고 싶은 내용]
.

**Comment by Tanney-102** [comment]
198 페이지에 나와있는 캐시 제어 플로우에는 웹 캐시에서 캐시여부를 제어하는 것으로 나타내는데 브라우저 캐시도 같은 과정을 브라우저가 주체적으로 실행하는 걸까요?

**Reply by ritajeong** [reply]
https://developer.mozilla.org/ko/docs/Web/HTTP/Caching

**Reply by ritajeong** [reply]
https://toss.tech/article/smart-web-service-cache

**Comment by Tanney-102** [comment]
cache control을 할때 로컬에 뭘 저장할까?

**Reply by ritajeong** [reply]
https://developers.google.com/web/fundamentals/instant-and-offline/web-storage?hl=ko

**Reply by Tanney-102** [reply]
https://mygumi.tistory.com/275
